prowler/providers/azure/services/network/network_service.py

```diff
@@ -0,0 +1,131 @@
+from dataclasses import dataclass
+
+import requests
+from azure.mgmt.network import NetworkManagementClient
+from azure.mgmt.network.models import NetworkWatcher
+
+from prowler.lib.logger import logger
+from prowler.providers.azure.lib.service.service import AzureService
+
+
+########################## SQLServer
+class Network(AzureService):
+    def __init__(self, audit_info):
+        super().__init__(NetworkManagementClient, audit_info)
+        self.token = self.__get_token__(audit_info)
+        self.security_groups = self.__get_security_groups__(self.token)
+        self.bastion_hosts = self.__get_bastion_hosts__()
+
+    def __get_security_groups__(self, token):
+        logger.info("SQL Server - Getting Network Security Groups...")
+        security_groups = {}
+        for subscription, client in self.clients.items():
+            try:
+                security_groups.update({subscription: []})
+                security_groups_list = client.network_security_groups.list_all()
+                available_locations = {}
+                network_watchers = self.__get_network_watchers__(client, subscription)
```

Comment:
Please, do a separate function and class for Network Watchers.

Reply:
Done